# Bepaling van soortelijke warmte van water


# Introductie

Water heeft een enorme buffercapaciteit: je moet veel energie toevoegen om de temperatuur van water een graad te verwarmen. In dit practicum gaan we de soortelijke warmte van water bepalen door een bekende hoeveelheid water te verwarmen met een bekende hoeveelheid energie, en de temperatuurstijging te meten.

# Theorie

Geef hier de natuurkundige achtergrond van het onderwerp. 

# Methode en materialen

## Ontwerp
Een waterbad met bekende massa aan water wordt verwarmd met een elektrisch verwarmingselement dat een bekende hoeveelheid energie levert.
De temperatuur van het water wordt gemeten met een temperatuursensor.
Door de temperatuurstijging als functie van de tijd te meten kan de soortelijke warmte van water worden berekend.


## Materialen
Hieronder staat de lijst van benodigde materialen bij deze proef:
- Maatbeker
- Weegschaal
- Water
- Elektrisch verwarmingselement ($10 \mathrm{\Omega}$, $10 \mathrm{W}$)
- Voedingsbron 
- Thermometer of temperatuursensor
- Stopwatch of timer

```{figure} figures/cwater_setup.jpg
:width: 70%

Een schematische weergave van de opstelling
```

## Procedure
```{exercise}
1. Beschrijf de procedure die nodig is om de soortelijke warmte van water te bepalen.
2. Geef jouw beschrijving van de procedure aan een ander team, zij verzamelen voor jou de data op basis van de gegevens.
3. Voer het experiment van een ander team uit op basis van hun beschrijving.
4. Analyseer de data die het andere groepje voor jou heeft verzameld.


Vul de maatbeker met 400ml water. Meet de starttemperatuur van het water. Verhit het water met het elektrische verwarmingselement tot ongeveer 5 graden boven de starttemperatuur en meet van begin tot eind van het verwarmen de tijd. Blijf het water de hele tijd roeren.  Bereken de toegevoerde warmte met Q=P*t. Bereken de soortelijke warmte met c =(P*t)/(m*deltat)
```


## Veiligheid
We maken gebruik van een $10 \mathrm{\Omega}$, $10 \mathrm{W}$ weerstand.
Deze wordt snel heet.
De bronspanning mag dan ook alleen aan wanneer de weerstand in het water zit.
Raak de weerstand niet aan tijdens het experiment.
Omdat de weerstand in het water zit, kunnen we wel het elektrisch vermogen hoger zetten zonder dat de weerstand oververhit raakt.
Het maximaal vermogen mag $40 \mathrm{W}$ zijn.
Daarbij moet de roerder wel aanstaan om de warmte goed te verdelen.

## Data analyse
Geef kort de data-analysemethode weer.

# Resultaten


In [ ]:
# Hier de data en de analyse
temp_begin = 23.2 # graden celcius
temp_einde = 26.8 # graden celcius
delta_temp = temp_einde - temp_begin
spanning = 15.4 # volt
stroom = 1.59  # ampere
tijd =  300 # seconde
massa = 0.4 #kg 
vermogen = spanning * stroom

c = (vermogen*tijd)/(massa*delta_temp)

print(c) 

5101.249999999998


In [ ]:
# Sla figuren op met  
# 
# plt.savefig("figures/naam.png", dpi=450)



# Discussie en conclusie

Hier een korte discussie en conclusie over de resultaten van het experiment en de implicaties daarvan.

Het resultaat van de vraag wat de soortelijke weerstand van water is 5101 J/kg*K. Dat komt redelijk overeen met de theroetische waarde maar is niet heel nauwkeuring. Dat komt door foute in de uitvoering van het experimenten en fouten in de onzekerheden. Een voorbeeld hiervan is dat de warmte bron niet een perfect constant vermogen geeft. Ook versliest het water ook warmte tijdens de proef. Onze conclusie is dus dat onze gevonde waarde van 5101 J/kg*K een goede benadering is maar dat er eenvervolg onderzoek moet worden uitgevoerd om een betere uitkomt te krijgen. In dat vervolg onderzoek moet er rekening gehouden worden met de limieten onze proef.